pkg/cmd/kafka/create/create.go

```diff
@@ -509,15 +511,22 @@ func promptKafkaPayload(opts *options, constants *remote.DynamicServiceConstants
 		Help:    f.Localizer.MustLocalize("kafka.create.input.name.help"),
 	}
 
-	// Get the list of enterprise clusters in the users organization if there are any
-	err := setEnterpriseClusterList(opts)
+	answers := &promptAnswers{}
+
+	err := survey.AskOne(promptName, &answers.Name, survey.WithValidator(validator.ValidateName), survey.WithValidator(validator.ValidateNameIsAvailable))
+	if err != nil {
+		return nil, err
+	}
+
+	// Get the list of enterprise clusters in the users organization if there are any, creates a map of cluster ids to names to include names in the prompt
+	err = setEnterpriseClusterList(opts)
```

Comment:
The https://github.com/redhat-developer/app-services-cli/blob/d9826b3b71bf87661a7a5cd04d842d50029dfb68/pkg/cmd/kafka/create/create.go#L460 needs to handle the `403` error being returned when making a call to KFM to list the clusters.
The error could be returned when the user doesn't not have quota. In that case, we should propose the normal flow as well.

Reply:
Done